hunts/src/api.js

```diff
@@ -21,12 +21,12 @@ function handleErrors(response) {
 }
 
 function getPuzzles(huntId) {
-  const puzzlesApiUrl = `/api/v1/hunt/${huntId}/puzzles`;
+  const puzzlesApiUrl = `/api/v1/hunts/${huntId}/puzzles`;
```

Comment:
Yes. Actually I think the URL scheme is kind of bad, there's no need for the /hunts/:id/ prefix on most of the endpoints

Reply:
I'm gonna update this in another PR after this because otherwise I think the merge conflicts will get a little messy. But I think I want to move:

/hunts/:id/puzzles/:id/tags => /puzzles/:id/tags
/hunts/:id/puzzles/:id/tags/:id => /puzzles/:id/tags/:id
/hunts/:id/puzzles/:id/answers => /puzzles/:id/answers
/hunts/:id/puzzles/:id/answers/:id => /puzzles/:id/answers/:id

Hopefully that's a little more readable. Then I'll change this api.js file to use a common /api/v1/ prefix

Reply:
Ok, that's fine with me